wiki/jee.md

```diff
@@ -361,6 +363,102 @@ In order to do this, you must provide the following content in the
 
 ... here, the `<class-loader delegate="false" />` part is the relevant bit.
 
+### Deactivate MoxyJson
+
+Apparently GF4 ships with MoxyJson which may cause problems like this, when you try to parse JSON within your application.
```

Comment:
It's Glassfish Payara 4.1 specific as far as I can tell because it comes with Moxy Json.

80 chars - will do :)